test/ModelTraitTest.php

```diff
@@ -49,23 +54,31 @@ public function testSetData($testData, $expectedProperties)
                 if (!empty($value["objectData"][0])) {
                     foreach ($value["objectData"] as $i => $objData) {
                         foreach ($objData as $subProp => $subValue) {
-                            $this->assertAttributeEquals($subValue, $subProp, $actualData[$i]);
+                            $assertGetter = "get" . ucfirst($subProp);
+
+                            if (is_array($actualData[$i])) {
+                                $this->assertSame($subValue, $actualData[$i][$subProp]);
```

Comment:
This if statement should no longer be required

Reply:
Removed